packages/edit-post/src/components/layout/style.scss

```diff
@@ -1,69 +1,107 @@
-$resize-handle-height: $grid-unit-30;
-
 .edit-post-meta-boxes-main {
 	filter: drop-shadow(0 -1px rgba($color: #000, $alpha: 0.133)); // 0.133 = $gray-200 but with alpha.
+	// Windows High Contrast mode will show this outline, but not the shadow.
+	outline: 1px solid transparent;
 	background-color: $white;
 	clear: both; // This is seemingly only needed in case the canvas is not iframe’d.
+	display: flex;
+	flex-direction: column;
+	overflow: hidden;
 
-	&:not(details) {
-		padding-top: $resize-handle-height;
-	}
-
-	// The component renders as a details element in short viewports.
-	&:is(details) {
-		& > summary {
-			cursor: pointer;
-			color: $gray-900;
-			background-color: $white;
-			height: $button-size-compact;
-			line-height: $button-size-compact;
-			font-size: 13px;
-			padding-left: $grid-unit-30;
-			box-shadow: 0 $border-width $gray-300;
-		}
-
-		&[open] > summary {
-			position: sticky;
-			top: 0;
-			z-index: 1;
-		}
+	&.is-resizable {
+		padding-block-start: $grid-unit-30;
 	}
 }
 
-.edit-post-meta-boxes-main__resize-handle {
+.edit-post-meta-boxes-main__presenter {
+	flex-shrink: 0;
```

Comment:
I don't think it's necessary since the parent element is not flex.

Reply:
Good spot! It’s still useful for the toggle button so I moved the style there 3f7553bb508168120a4116d07b93e7cf108ea0e8. It was in play for the separator button when `ResizableBox` wasn’t being used.